nucliadb_vectors2/src/vectors/indexset/mod.rs

```diff
@@ -18,65 +18,66 @@
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 
 mod state;
-
-use std::mem;
 use std::path::{Path, PathBuf};
 use std::sync::RwLock;
 
 use state::State;
 
-use crate::disk::directory;
-use crate::disk::directory::{ELock, Lock, SLock, Version};
+use crate::disk::directory::{self, ELock, Lock, SLock, Version};
 use crate::vectors::data_point_provider::{Index, VectorR};
 
 pub trait IndexKeyCollector {
     fn add_key(&mut self, key: String);
 }
 
+#[derive(Clone, Copy, Debug)]
+pub enum IndexSetCheck {
+    None,
+    Sanity,
+}
+
```

Comment:
Is it really useful to duplicate this enum ? I mean `IndexCheck`  can be used for both cases

Reply:
done.